portfolio/src/main/webapp/script.js

```diff
@@ -59,7 +59,9 @@ function toggleBlogPost(buttonId) {
  * @return none
  */
 function fetchComments() {
-  fetch('/data').then(response => response.json()).then((commentData) => {
+  let quantity = 3;
+  var queryString = '/data' + '?quantity=' + String(quantity);
```

Comment:
This is OK for now, but I'm starting to get a little nervous that you're generating the URIs manually.

https://developer.mozilla.org/en-US/docs/Web/API/URL -> http://go/mdn/API/URLSearchParams has an append() method, so maybe that's what you need to use as things get more complicated?

Reply:
I have refactored the URL building to use URLSearchParam to make it easier to add more params in the future, if need be 👍🏻